samples/tool/SFSClientTool.cpp

```diff
@@ -382,23 +414,40 @@ Result GetLatestDownloadInfo(const SFSClient& sfsClient, const Settings& setting
             return result.GetCode();
         }
 
-        // Display results
-        DisplayResults(appContents);
+        out = AppContentsToJson(appContents);
     }
     else
     {
         std::vector<Content> contents;
         auto result = sfsClient.GetLatestDownloadInfo(params, contents);
-
         if (!result)
         {
             PrintError("Failed to get latest download info.");
             LogResult(result);
             return result.GetCode();
         }
 
-        // Display results
+        out = ContentsToJson(contents);
+    }
+
+    return Result::Success;
+}
+
+Result HandleJSONContents(const Settings& settings, const json& contents)
+{
+    if (settings.outputFile.empty())

```

Comment:
nit? I'd add a function that does this to make intent clearer, e.g.

`bool settings::OutputToFile() const { return !outputfile.empty(); }`

Reply:
and do if (OutputtoFile()) { ... jsonto file... }

avoids a "not (.empty)" and a "outputfile.empty" means "not to file"